docs/aerodromes/Class-C/NZAA/index.md

```diff
@@ -4,113 +4,89 @@
 
-Arrivals on a straight-in approach - at LENGU or EMRAG
+Pushback and Start up clearances are managed by the Ground Controller. Pilots can expect to recive a "Tail Facing" direction with a pushback clearance (doesnt normally include aircraft from regional gates). This can be to facilitate other departures or arrivals, with the layout of the taxiways, pilots are generally assumed to not block other taxiways.
 
-Arrivals on a visual approach  - by 15 AA DME
+### Domestic 
 
-Arrivals on an RNP approach, not straight-in - at ATAMA or NIPAS (RWY05R), ESNAX, ESMON, TEGIK, or TUSLO (RWY23L).
+Aircraft pushing back from gates 20-22 normally will push back onto D1/D2 depending on other movements or incases onto B. Aircraft on 24 can be pushed back to whereever the controller sees suitable at the time. Aircraft on 28-33 shall generally be pushed facing the direction of the active runway, although given the layout that can change. 
 
----
+For stands 28-33 there may be multiple aircraft pushing at once and along with arriving traffic, taxiways like B4, B5 and B6 tend to be kept clear for other aircraft. This means a tail facing direction should be given to ensure positive control. Aircraft can also be told to push clear of a taxiway eg. B5 to facilitate another aircraft arriving or departing. For example and aircraft may push off 30 and tail to the west clear of B5, while an aircraft of 31 may push tail east clear of B5. Therefore there is equal room for either aircraft to get out. Aircraft of 32 and 33 may be pushed down to the east side of B4 to facilitate other traffic at the regional gates etc. 
 
-## Auckland Ground
+Aicraft on C5 requiring a push clearance will be given a "push Charlie five your discretion" clearance since it is not apart of the manourvering area, but still requires a clearance to push. 
 
-Auckland Ground is responsible for issuing taxi instructions within the movement area and providing separation to all aircraft on the taxiways at Auckland Airport.
+### International 
 
-### Pushback & Movement Areas
+At international gates things are more simple, aircraft pushing off of any gate at international are given a tail facing direction and a start location. For example: "Push tail north start abeam stand 15". Aircraft tend to be towed forward but we will ignore this. 
 
-Auckland Ground issues pushback and start-up advice/clearances to all departures. 
+## Taxiing 
 
-Ground uses positive control to ensure that no other aircraft passes directly behind an aircraft pushing back, even when the aircraft concerned are not on the controlled movement area, this includes the C aprons, such as C5.
+With the way the taxiways are layed out at Auckland, controllers avoid using taxiway A where needed. When RWY 23L is active, domestic aircraft may be taxied where they need as they dont interat with rapid exits like A6/A8. In high traffic loads or to aliviate delays international aircraft are taxied down via B and B6 instead of all the way along A. 
```

Comment:
```suggestion
With the way the taxiways are laid out at Auckland, controllers avoid using taxiway A where needed. When RWY 23L is active, domestic aircraft may be taxied where they need as they don't interact with rapid exits like A6/A8. In high traffic loads or to alleviate delays international aircraft are taxied down via B and B6 instead of all the way along A. 
```